npm/pkg/controlplane/controllers/v1/networkPolicyController.go

```diff
@@ -245,6 +245,13 @@ func (c *NetworkPolicyController) syncNetPol(key string) error {
 	if err != nil {
 		if errors.IsNotFound(err) {
 			klog.Infof("Network Policy %s is not found, may be it is deleted", key)
+
+			if _, ok := c.rawNpMap[key]; ok {
+				// record time to delete policy if it exists (can't call within cleanUpNetworkPolicy because this can be called by a pod update)
+				timer := metrics.StartNewTimer()
```

Comment:
Again, we can put this at the top of the clean up and syncAdd functions, that will reduce the complexity of adding it in multiple places.

Reply:
synced offline about keeping as is